test/Microsoft.AspNetCore.Razor.Evolution.Test/DefaultRazorParsingPhaseTest.cs

```diff
@@ -46,6 +46,46 @@ public void Execute_UsesConfigureParserFeatures()
             Assert.Equal("test_directive", directive.Name);
         }
 
+        [Fact]
+        public void Execute_ParsesIncludesAndImports()
+        {
+            // Arrange
+            var phase = new DefaultRazorParsingPhase();
+            var engine = RazorEngine.CreateEmpty((b) =>
+            {
+                b.Phases.Add(phase);
+                b.Features.Add(new MyConfigureParserOptions());
+            });
+
+            var imports = new[]
+            {
+                TestRazorSourceDocument.Create(),
+                TestRazorSourceDocument.Create(),
+            };
+
+            var includes = new[]
+            {
+                TestRazorSourceDocument.Create(),
+                TestRazorSourceDocument.Create(),
+            };
+
+            var codeDocument = TestRazorCodeDocument.Create(TestRazorSourceDocument.Create(), imports, includes);
+
+            // Act
+            phase.Execute(codeDocument);
+
+            // Assert
+            Assert.Collection(
```

Comment:
Eh, this is a bit too simple to assume that the import/include trees are being parsed independently.  I'd add a single line content to each and verify that we get blocks/spans etc. for them.